components/chains-list.tsx

```diff
@@ -35,74 +35,48 @@ const ChainsList = () => {
 
   useEffect(() => {
     if (userChainId && supportedChains.includes(userChainId)) {
-      setSelectedChain(userChainId);
       setIsChainValid(true);
     } else {
-      setSelectedChain(sepolia.id);
       setIsChainValid(false);
     }
   }, [userChainId]);
 
   return (
-    <section>
-      {isConnected && !isChainValid && (
-        <Card className="relative z-20 mb-4 flex flex-col items-center gap-4 border-none bg-amber-100 p-8 text-center">
-          <h2 className="text-xl font-semibold">
-            We dont support the chain you are currently connected to just yet 🫥
-          </h2>
-          <p className="text-xl font-normal">
-            Check out the jars on our other chains 👇
-          </p>
-        </Card>
-      )}
-      {(!isConnected || !isChainValid) && (
-        <div className="mb-4">
-          <ChainsSelector
-            selectedId={selectedChain}
-            setSelectedId={setSelectedChain}
-          />
+    <Card className="relative z-20 flex flex-col items-center gap-8 border-none bg-amber-100 p-8 text-center">
+      <h1 className="font-gluten text-5xl font-semibold">Jars</h1>
+      <div className="mx-2 flex w-full gap-4 px-4">
+        <select
+          className="text-strong rounded-md bg-amber-200 p-2 font-bold"
+          onChange={(val) =>
+            changeSorting(val.target.value as SortingSelectValues)
+          }
+        >
+          <option value="cookieAmount-desc">Largest cookie stash</option>
+          <option value="cookieAmount-asc">Smallest cookie stash</option>
+          <option value="periodLength-asc">Shortest claim period</option>
+          <option value="periodLength-desc">Longest claim period</option>
+        </select>
```

Comment:
Use existing Select component found in @/components/ui/select
```suggestion
        <select
          className="text-strong rounded-md bg-amber-200 p-2 font-bold"
          onChange={(val) =>
            changeSorting(val.target.value as SortingSelectValues)
          }
        >
          <option value="cookieAmount-desc">Largest cookie stash</option>
          <option value="cookieAmount-asc">Smallest cookie stash</option>
          <option value="periodLength-asc">Shortest claim period</option>
          <option value="periodLength-desc">Longest claim period</option>
        </select>
```